packages/client/vite.config.js

```diff
@@ -15,6 +15,7 @@ import tsconfigPaths from 'vite-tsconfig-paths'
 
 process.env.VITE_APP_COUNTRY_CONFIG_URL =
   process.env.COUNTRY_CONFIG_URL || 'http://localhost:3040'
+process.env.VITE_APP_VERSION = process.env.VITE_APP_VERSION || 'Development'
```

Comment:
We don't need to add this here. The `COUNTRY_CONFIG_URL` was needed because it doesn't start with `VITE`(vite ignores the env variables that doesn't start with it)

Reply:
@Zangetsu101 the rationale behind this was to add a default value for the environment variable someway 🤔 I wonder if there's another way to achieve that or would this be good?